docs/lib/project-setup/fragments/ios/create-application/20_install_libraries.md

```diff
@@ -0,0 +1,86 @@
+<amplify-block-switcher>
+
+<amplify-block name="Swift Package Manager">
+
+To start adding the Amplify Libraries to your iOS project, open your project in Xcode and select **File > Swift Packages > Add Package Dependency**.
+
+![Add package dependency](~/images/project-setup/20_4_add-package-dependency.png)
+
+Enter the Amplify iOS GitHub repo URL (`https://github.com/aws-amplify/amplify-ios`) into the search bar and click **Next**.
+
+![Search for repo](~/images/project-setup/20_5_search-amplify-repo.png)
+
+You'll see the Amplify iOS repository rules for which version of Amplify you want Swift Package Manager to install. Choose the first rule, **Version**, as it will use the latest compatible version of the dependency that can be detected from the `main` branch, then click **Next**.
+
+![Dependency version options](~/images/project-setup/20_6_dependency-version-options.png)
+
+Lastly, choose which of the libraries you want added to your project. Always select the **Amplify** library. The "Plugin" to install depends on which categories you are using:
+
+- API: **AWSAPIPlugin**
+- Analytics: **AWSPinpointAnalyticsPlugin**
+- Auth: **AWSCognitoAuthPlugin**
+- DataStore: **AWSDataStorePlugin**
+- Storage: **AWSS3StoragePlugin**
+
+  _Note: AWSPredictionsPlugin is not currently supported through Swift Package Manager due to different minimum iOS version requirements. Support for this will eventually be added._
```

Comment:
nit: `<amplify-callout>` could be used depending on the level of emphasis needed here